### PR TITLE
Consistency check done between README and mockingcase.js

### PR DESCRIPTION
**Pull request by Archulan**
I have added the recommendation you made for the last pull request

### CODE REVIEW

File: src/mockingcase.js [hunk 1]
```diff
@@ -140,8 +142,8 @@ function isArrayOfStrings(input) {
 
   input.forEach((value, i) => {
     if (typeof value !== "string") {
-      throw TypeError(`Expected array of strings but got type '${typeof value}' at index ${i}`);
-    }
+    throw TypeError(`Expected array of strings but got type '${typeof value}' at index ${i}`);
+  }
```
Comment (strdr4605) [inline src/mockingcase.js]: Please don't remove indentation.

Comment (Archulan) [reply src/mockingcase.js]: okay

**Comment by Archulan** [comment]
> LGTM

Okay @strdr4605. I would like to work on React Native related repositories using Javascript. Recommend me if any.

**Comment by strdr4605** [comment]
@Archulan, there is this package https://github.com/junedomingo/react-native-rename. Because of my project requirements, I am using it a lot. But the problem is that is not really maintained. I forked it and created https://github.com/strdr4605/react-native-rename which is published https://www.npmjs.com/package/@strdr4605/react-native-rename.

I implemented:
https://github.com/junedomingo/react-native-rename/pull/93
and
https://github.com/junedomingo/react-native-rename/issues/95 at https://github.com/strdr4605/react-native-rename/blob/master/src/index.js#L254
this is not the best solution, maybe you can work on refactoring it.
Also, another interesting issue is https://github.com/junedomingo/react-native-rename/issues/94.
You can fork my fork and create PR but also in the original repo.

**Pull request merged**